default.yaml

```diff
@@ -25,16 +25,17 @@ hedging:
   HIGH_SAFEBOUND_LEVERAGE: 2.25
   HIGH_BOUND_LEVERAGE: 3
 
-rebalancing:
+coldStorage:
   minOnChainHotWalletBalance: 1000000 # 0.1
   minRebalanceSize: 10000000 # 0.1
   maxHotWalletBalance: 200000000 # 2
-
-  # select which bitcoind wallet is used for rebalancing
+  # pattern used to load cold storage wallet balances
   # string.include() is being used as a filter
-  #
-  # specter wallet always
-  onchainWallet: "specter"
+  walletPattern: "specter"
+
+  # select which bitcoind wallet is used for automatic rebalancing
+  onChainWallet: "specter/coldstorage"
+  targetConfirmations: 6
```

Comment:
a comment on what this `targetConfirmations` is for would be useful

Reply:
Added https://github.com/GaloyMoney/galoy/pull/986/commits/b63f45d6355724f67e023865cb04742b9eb02b50#diff-94d4d5bb5c048793536625edb8adffb17e89b5f3821cec6d1c7c337578cda8e4R40